runners/java-fn-execution/pom.xml

```diff
@@ -63,11 +63,6 @@
       <artifactId>beam-runners-core-construction-java</artifactId>
     </dependency>
 
-    <dependency>
-      <groupId>com.google.protobuf</groupId>
```

Comment:
Note this was deleted because it was repeated in the same file as well as core-construction-java.